http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -57,8 +57,11 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
       )
 
       // Databases required for Cromwell App
-      dbNames <- F.fromOption(toCromwellRunnerAppDatabaseNames(params.databaseNames),
-                              AppCreationException("Database names required for Cromwell Runner app", Some(ctx.traceId))
+      dbNames <- F.fromOption(
+        toCromwellRunnerAppDatabaseNames(params.databaseNames),
+        AppCreationException(s"Database names required for Cromwell Runner app: ${params.databaseNames}",
```

Comment:
I found this update to the log message helpful, and I thought it was worth keeping (unlike many of my other added logs, now removed)